src/ui/components/EamlightToolbar.js

```diff
@@ -138,41 +142,80 @@ class EamlightToolbar extends Component {
+    
+            if (!tabs[category]) {
+                tabs[category] = [];
+            }
+    
+            if (!region.ignore) {
+                let item = (
+                    <MenuItem key={region.id} onClick={() => toggleHiddenRegion(getUniqueRegionID(region.id))}>
+                        <Checkbox disabled checked={!isHiddenRegion(region.id)} />
+                        {region.label}
+                    </MenuItem>
+                );
+                tabs[category].push(item);
+            }
+        }
+    
+        return (
+            <>
+                {Object.keys(tabs).map((category, index) => (
+                    <div key={index}>
+                        <ListItemButton
+                            onClick={ () => {
+                                this.setState(
+                                    prevState => ({
+                                        collapsibleMenu: {
+                                            ...prevState.collapsibleMenu,
+                                            [category]: !prevState.collapsibleMenu[category],
+                                        }
+                                    })
+                                );
+                            }}
+                        >
+                            {category} {this.state.collapsibleMenu[category] ? <ExpandLess /> : <ExpandMore />}
+                        </ListItemButton>
+                        <Collapse in={this.state.collapsibleMenu[category]}>
+                            {tabs[category]}
+                        </Collapse>
+                    </div>
+                ))}
+            </>
+        );
     }
-
-    //
-    //
-    //
-
+    
     renderPanelSelectorMenu = (isInsideAMenu = false) => {
         return (
-            <div style={{flexGrow: '1'}}>
-                {isInsideAMenu && <Divider/>}
+            <div style={{ flexGrow: '1' }}>
+                {isInsideAMenu && <Divider />}
                 <IconButton
                     aria-label="More"
                     aria-owns={this.state.visibilityMenu ? 'simple-menu' : null}
                     onClick={this.handleVisibilityMenuClick.bind(this)}
-                    size="large">
-                    <TelevisionGuide style={isInsideAMenu ? {width: 18, marginRight: -5, marginLeft: 5} : {}}/> 
+                    size="large"
+                >
+                    <TelevisionGuide style={isInsideAMenu ? { width: 18, marginRight: -5, marginLeft: 5 } : {}} />
                 </IconButton>
                 {isInsideAMenu && <span>Panel Selector</span>}
                 <Menu
                     id="simple-menu"
                     anchorEl={this.state.visibilityMenu}
                     open={Boolean(this.state.visibilityMenu)}
                     onClose={this.handleVisibilityMenuClose.bind(this)}
+                    PaperProps={{
+                        style: {
+                            maxHeight: '75vh',
```

Comment:
To be tested on a mobile version, because sometimes the ‘vh’ unit doesn't perform well on a small screen